### ISSUE TITLE
Profile detail (graph) page is not available with IndexError

### ISSUE THREAD

**Issue by youknowone** [amcfague/linesman#30]
Error log
```
127.0.0.1 - - [28/Aug/2013 15:52:42] "GET /__profiler__/profiles/30f64b40-0fae-11e3-b925-c42c030096d3 HTTP/1.1" 500 -
Traceback (most recent call last):
  File "<path>/.env/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "<path>/.env/lib/python2.7/site-packages/linesman/middleware.py", line 136, in __call__
    wsgi_app = self.show_profile(req)
  File "<path>/.env/lib/python2.7/site-packages/linesman/middleware.py", line 312, in show_profile
    session._graph, cutoff_time, True)
  File "<path>/.env/lib/python2.7/site-packages/linesman/middleware.py", line 441, in prepare_graph
    u, v = cycle[0], cycle[1]
IndexError: list index out of range
```

environment:
- OS X 10.8
- graphviz with homebrew (graphviz: stable 2.32.0, devel 2.33.20130804.0447)

pip freeze:
```
Flask==0.10.1
linesman==0.3.1
```

**Comment by LuisUrrutia** [comment]
+1 with the issue and fix (networkx==1.7)

**Comment by amcfague** [comment]
I apologize this went on for so long without my addressing it--I'm not sure why I didn't see this issue.

I'm at work and won't be able to patch this (maybe tonight, if I have time).  If someone wanted to submit a pull request, I could easily do it from here.

**Comment by amcfague** [comment]
Released under version 0.3.2.
